components/fxa-client/src/http_client.rs

```diff
@@ -660,6 +706,35 @@ pub struct DeviceResponseCommon {
     pub push_endpoint_expired: bool,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct GetAttachedClientResponse {
+    #[serde(rename = "clientId")]
+    pub client_id: String,
```

Comment:
You can avoid renaming most of these fields with `#[serde(rename_all = "camelCase")]` on the struct.